exec/Main.hs

```diff
@@ -23,12 +25,15 @@ main = do
     initializeTables conn
     putStrLn "DB Tables Initialized"
     m <- newManager tlsManagerSettings
-    let !env = Env m pgc u v
-    case c of
-      Listen -> listen env
-      Backfill -> backfill env
-      Gaps -> gaps env
-      Single cid h -> single env cid h
+    allChains m u >>= \case
+      Nothing -> printf "[FAIL] Unable to contact %s for Chain counts\n" u >> exitFailure
```

Comment:
While we're changing the  above, let's make this a little more accurate.  Something like `Unable to connect to node`.

Reply:
The `u` is the address of the Node, or is that still not clear enough?

Reply:
Eh...I think the "for chain counts" is a little too specific.